### PR TITLE
Fixing issue in meshery.io, issue #562

### LINKED ISSUE #562: Button in a Button Again
**Description**
I think someone has edited the div of the button or the button and the classes applied to different pages are facing this button in a button issue. 

**Expected Behavior**
Currently only this page buttons needs to be updated asap.

**Screenshots**

**Enviroment:**
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

---
[Optional] **To Reproduce**

[Optional] **Additional Context**

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by AndrewArnita**
Signed-off-by: Andrew <andrew_arnita@hotmail.com>

**Description**

This PR fixes #562 

**Notes for Reviewers**
This is my first contribution to a project!

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ter [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (warunicorn19) [inline ter:1]: I don't think we need this file in the commit.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Other changes look good to me, if you could just revert the file that I mentioned.

**Comment by AndrewArnita** [comment]
Hello @warunicorn19, please if you can check my new commit, I removed the ter file

**Comment by warunicorn19** [comment]
The ter file still seems to be there in the 'files changed' I think you removed the package-lock.json.

**Comment by warunicorn19** [comment]
Good to go now, but you forgot to sign-off the last commit😅.  Try following this https://github.com/meshery/meshery.io/pull/563/checks?check_run_id=3697665094 to sign-off now

**Comment by AndrewArnita** [comment]
@warunicorn19 thank you for your patience, I am new here so I am learning how things work and how to deal with git revert. I think now there are only 2 file changes (list-posts.html & package-lock.json). I still have to sign off my commits and I am good to go :)

**Comment by warunicorn19** [comment]
No problem @AndrewArnita 😅, please take your time with it 😃.

**Comment by AndrewArnita** [comment]
@warunicorn19, everything should be working perfectly now! I just signed off all my commits :)

**Review by warunicorn19 — DISMISSED** [review]
Looking good, thanks @AndrewArnita for this.😅

**Comment by AndrewArnita** [comment]
This issue is done from my part! Thanks @warunicorn19 for your help :)

**Comment by warunicorn19** [comment]
@AndrewArnita I think it'd be better if you create a new PR, everytime you force push to sign off the `ter` file will get added to the commit. So, I'd suggest you to create a new branch and make a new PR. Before creating a new branch remember to switch to the master branch first `git switch master`

**Comment by AndrewArnita** [comment]
Okay @warunicorn19, I will work on it!

**Pull request closed without merging**